BP/common/culture/cultures/00_central_germanic.txt

```diff
@@ -0,0 +1,1737 @@
+			{ "dynnp_van" "dynn_Diest" }
+			{ "dynnp_van" "dynn_Voorne" }
+			{ "dynnp_van" "dynn_Naaldwijk" }
+			{ "dynnp_de" "dynn_Moravia" }
+			{ "dynnp_van" "dynn_Yperen" }
+			{ "dynnp_van" "dynn_Zeeland" }
+			"dynn_Munthe"
+			"dynn_Mard-e_Koochak"
+			"dynn_Hoving"
+			"dynn_Kuiper"
+		}
+		
+		male_names = {
+			Adriaen Albrecht Alwin Ambroos Antoon Arnold Arnulf Balderik Barend Barnaerd Bartolomeus
+			Bavo Bertold Boudewijn Claes Claeuwaert Daevit Damaes Diederick Dirk Douwe Eduard Egmund Ekbert
+			Emmo Engelbrecht Filips Floris Folbert Frans Frederik Gerard Gerolt Gerulf
+			Gerrit Gijsbrecht Giselbert Godfried Godschalk Gozewijn Guij Gwijde Hendrik Herman Hubert
+			Hugo Ijsbrand Ijsbrecht Jacco Jacob Jan Karel Koenraad
+			Kristof Lambert Lodewijk Ludewijn Lutbert Luthard Maertijn Marquart Niclaes
+			Olivier Otto Otwin Onno Pieter Reinaerd Reinoud Reginar Robrecht Robert Rodulf
+			Roelof Roland Rutger Sebastiaen Sicco Simon Steven Steyn Theodolf Waelram Walter
+			Wicher Willem Wolfard
+		}
+		female_names = {
+			Ada Adela Adelheid Aechte Agaete Aleid Alijse Aliken Anna Annelies Badeloge Bava Beatrijs Benedicta
+			Blanca Blanceflore Cecily Cille Clara Clementia Elisabeth Ellijnore Emma Ermingarde Fenne
+			Filippa Frida Gheertrude Gerda Godila Haduwijn Helena Herlinda Hilligonda Ide Imagina Irene Isabelle Iudith
+			Jacoba Jaccobijn Jasminne Johanna Jolanda Juliana Jutte Kathrijne Katrien Liedwin Lisebet Machtelt Margareta
+			Maria Mathilde Oda Odelt Olinde Onna Pieternelle Reynilde Sanne Sigerda Sophie Swanhildis Thetburga Waltrud
+			Wila Wendelmoet Wenna Willemine Yonefij Yvonne Zoete
+		}
+		
+		dynasty_of_location_prefix = "dynnp_van"
+		
+		patronym_suffix_male = "dynnpat_suf_szoon"
+		patronym_suffix_female = "dynnpat_suf_sdochter"
+		always_use_patronym = yes	
+		
+		# Chance of male children being named after their paternal or maternal grandfather, or their father. Sum must not exceed 100.
+		pat_grf_name_chance = 30
+		mat_grf_name_chance = 10
+		father_name_chance = 25
+		
+		# Chance of female children being named after their paternal or maternal grandmother, or their mother. Sum must not exceed 100.
+		pat_grm_name_chance = 20
+		mat_grm_name_chance = 40
+		mother_name_chance = 5
+
+		ethnicities = {
+			25 = caucasian_blond
+			15 = caucasian_ginger
+			35 = caucasian_brown_hair
+			25 = caucasian_dark_hair
+		}
+
+		mercenary_names = {
+			{ name = "mercenary_company_of_the_brabancon_wolf" coat_of_arms = "mc_company_of_the_brabancon_wolf" }
+			{ name = "mercenary_company_flemish_band" }
+			{ name = "mercenary_false_earls_company" }
+		}
+	}
+
+	# WTWSMS
+
+	german = {
+		color = { 0.49 0.41 0.24 }
+
+		cadet_dynasty_names = {
+			{ "dynn_Amaling" }
+			{ "dynn_Hasding" }
+		}
+		
+		dynasty_names = {
+			{ "dynn_Amaling" }
+			{ "dynn_Hasding" }
+		}
+
+		male_names = {
+			Adalbert_Albert Adam_Adam Adolf Albrecht_Albert Almerich_Amalric Amadeus Andreas_Andrew Anselm Arnold_Arnold Arnulf_Arnulf Augustin 
```

Comment:
Did you research a new name list for this or are you just replacing the CK3 name list with the one from WTWSMS? If it is the latter it is not necessary, I'm pretty sure the CK3 vanilla name list is a lot better than the old one.

Reply:
Took our namelist yeah though I changed it back but forgot the names (the most important part lol) but yeah I'll replace them.